altair/utils/mimebundle.py

```diff
@@ -147,7 +145,7 @@ def _validate_normalize_engine(engine, format):
 
     engine : {None, 'vl-convert', 'altair_saver'}
         the user-provided engine string
-    format : string {'png', 'svg', 'pdf', 'vega'}
+    format : string {'png', 'svg', 'pdf'}
```

Comment:
```suggestion
    format : string {'png', 'svg', 'pdf', 'vega'}
```
After your latest changes, format can be `'vega'`, right?